### ISSUE TITLE
Handle driver exceptions more elegantly

### ISSUE THREAD

**Issue by tjrivera** [chop-dbhi/ehb-datasources#8]
Currently comm issues with Nautilus will raise an Exception. These should be handled and appropriate feedback should be given to the user.

**Comment by tjrivera** [comment]
Naut-REST Errors to be mapped:

```
NAUTILUS_LOGIN_FAILURE = '1'
USERNAME_NOT_PROVIDED = '2'
PASSWORD_NOT_PROVIDED = '3'
REQUEST_TYPE_NOT_PROVIDED = '4'
REQUEST_BODY_NOT_PROVIDED = '5'
MALFORMED_REQUEST = '6'
UNSUPPORTED_REQUEST_TYPE = '7'
INVALID_LIMS_REQUEST_BODY = '8'
NAU_SOCKET_SERVICE_NOT_FOUND = '100'
NAU_INVALID_AUTH_HEADER= '101'
```

**Comment by seg1129** [comment]
issue being tested in dev

**Comment by seg1129** [comment]
Issue resolved in version 1.0.4
